### PR TITLE
Deprecate ImageToTensor in favor of vision transform

### PR DESCRIPTION
**Pull request by carmocca**

### CODE REVIEW

File: pylaia-htr-netout [hunk 1]
```diff
@@ -11,16 +11,74 @@ import torch
...
+            )
+            for row in range(output_size)
+            for col in range(output.size(1))
+        )
+        + "\n"
+    )
+    matrix += "{:d}\t0,0,\n".format(output_size)
+    return matrix
+
+
```
Comment (jpuigcerver) [inline pylaia-htr-netout]: This should not be here, they were substituted by ArchiveMatrixWriter and ArchiveLatticeWriter.

File: laia/data/transforms/vision/transforms.py [hunk 2]
```diff
@@ -33,4 +33,60 @@ def __repr__(self):
         return format_string
 
 
+class ToPreparedTensor(object):
```
Comment (jpuigcerver) [inline laia/data/transforms/vision/transforms.py]: This name is confusing. Can you think of a better name?

Comment (carmocca) [reply laia/data/transforms/vision/transforms.py]: Maybe ToImageTensor?

**Comment by carmocca** [comment]
This PR has been incorporated into my fork, which will be eventually merged upstream.

Closing this.

**Pull request closed without merging**